### PR TITLE
Update Classification.md

### PR DESCRIPTION
**Pull request by MarcNeely**
Add something about restrictions for volume classifiers.

### CODE REVIEW

File: docs/learning/display/Classification.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Spatial classification and planar clip masking are related ways of using one set
 Spatial classification uses one or more design models to contextualize a reality model by associating portions of the reality model with elements in the design models. Two types of classification are supported:
 
 - Planar classification: The design models are projected onto a plane. The region of the reality model that intersects the volume extruded from an element's footprint become classified by that element.
-- Volume classification: Regions of the reality model that intersect the 3d volume of an element become classified by that element.
+- Volume classification: Regions of the reality model that intersect the 3d volume of an element become classified by that element.  Geometry used for volume classification must be fully closed, non-intersecting volumes.
```
Comment (pmconne) [inline docs/learning/display/Classification.md:10]: Please add the same clarification to the [learning article](https://github.com/iTwin/itwinjs-core/blob/master/docs/learning/display/Classification.md).

**Review by pmconne — APPROVED** [review]

File: docs/learning/display/Classification.md (hunk 1 above)
Comment (MarcNeely) [reply docs/learning/display/Classification.md:10]: I thought that's what I just did.  I went to the learning article and used the edit command and after finishing it created this PR for it.

Comment (pmconne) [reply docs/learning/display/Classification.md:10]: I'm sorry -- I meant the [API documentation](https://github.com/iTwin/itwinjs-core/blob/aca5d1769dc2eb12e83bedf49e7e5804b9c33416/core/common/src/SpatialClassification.ts#L144).

Comment (MarcNeely) [reply docs/learning/display/Classification.md:10]: OK, Done.

**Review by markschlosseratbentley — APPROVED** [review]

**Comment by MarcNeely** [comment]
Hmmm, the PR that was created automatically when I edited the learning page isn't merging into master and is instead merging into release/3.2.x, so I'm abandoning this PR to start a new one from master.  This one is having a hard time passing the security audit from errors which have nothing to do with the files that I changed.

**Pull request closed without merging**